jina/peapods/pods/__init__.py

```diff
@@ -586,6 +587,49 @@ def _set_peas_args(
         return result
 
     def _parse_base_pod_args(self, args):
+
+        if getattr(args, 'uses', None):
+            # use the executor existed in Jina Hub.
+            if re.match(r'^jinahub(\+docker)?://', args.uses):
```

Comment:
please avoid homemade URL parsing and building, this can be easily broken.

```python
>>> import urllib.parse
>>> urllib.parse.urlparse('jinahub+docker://text-bert').scheme
'jinahub+docker'
>>> urllib.parse.urlparse('jinahub://text-bert').scheme
'jinahub'
>>> 

```